src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    static final int FIRST_INDEX = 0;
+    static final int SECOND_INDEX = 1;
+    static final int THIRD_INDEX = 2;
+    static final int FOURTH_INDEX = 3;
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
    private static final int WORK_DATE_INDEX = 0;
    private static final int NAME_OF_WORKER_INDEX = 1;
    private static final int HOURS_PER_DAY_INDEX = 2;
    private static final int SALARY_PER_HOUR_INDEX = 3;
```

FIRST_INDEX, SECOND_INDEX , .. not very informative names